src/Nox.Generator/Presentation/Api/OData/EntityController.Entity.template.cs

```diff
@@ -41,6 +41,9 @@ public abstract partial class {{entity.PluralName}}ControllerBase : ODataControl
     /// </summary>
     protected readonly IMediator _mediator;
 
+    /// <symmary>
+    /// The HTTP language provider.
+    /// </symmary>
     protected readonly Nox.Presentation.Api.IHttpLanguageProvider _httpLanguageProvider;
 
     public {{entity.PluralName}}ControllerBase(
```

Comment:
In the constructor, we can fetch the language:
`var language = httpLanguageProvider.GetLanguage();
_cultureCode = Nox.Types.CultureCode.From(language)`

Reply:
Done. Thanks for the suggestion.